### ISSUE TITLE
Transform A Property Page: Change card titles to H3s

### ISSUE THREAD

**Issue by marvieqa** [CodeForPhilly/clean-and-green-philly#618]
## Description
Card titles found in Transform Property page is marked up as H4s. These should be H3s so that we don’t skip heading levels.

## Acceptance Criteria
- [ ] Change all card titles to `<h3>`
  - [ ] Basic Intervention Card titles
  - [ ] Advanced Intervention Card titles

**Comment by CodeWritingCow** [comment]
Hello @marvieqa can you assign this ticket to me? Thank you!

**Comment by marvieqa** [comment]
> Hello @marvieqa can you assign this ticket to me? Thank you!

Done! Thanks @CodeWritingCow !
